build.psm1

```diff
@@ -20,7 +20,7 @@ try {
 }
 
 $dotnetCLIChannel = "preview"
-$dotnetCLIRequiredVersion = "2.0.0-preview1-005952"
+$dotnetCLIRequiredVersion = "2.0.0-preview2-006502"
```

Comment:
preview3 .NET CLI is still broken, please see the description of this PR.
Now we are using .NET CLI preivew2 to build, but target preview3 .NET Core Framework.